src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+@import './utils/variables';
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: $base-size * 7 + $base-gap * 6;
+  background-color: #fff;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $base-size;
+    width: $base-size;
+    border: 1px solid #000;
+    background-color: $day-color;
+    transition: all 0.5s;
```

Comment:
It's better to specify which properties you are transitioning rather than using 'all' for performance reasons. Specify the exact properties you want to transition, in this case, 'background-color' and 'transform'.